mgmt-hub/README.md

```diff
@@ -57,7 +57,7 @@ Then you can run these commands:
     "destinationType": "pattern-ibm.helloworld"
   }
   EOF
-  echo -e "foo\nbar" > mms-file
+  echo "foo\nbar" > mms-file
```

Comment:
This change isn't correct, because omitting `-e` will cause `echo` to not process the newline correctly (on both mac and linux).

On my mac, the `echo` command supports the `-e` flag correctly (even though `man echo` doesn't show it as a valid flag). What version of macos are you running?

Reply:
@bmpotter Thanks for your response here. you are right.  but I don't know how to check the  version of  `echo` which should be a built-in shell command. 
```
> which echo 
echo: shell built-in command.
```
so, after checking, it maybe caused by shell env which we are using. it always work well in bash environment:
```
> bash
engbuild@ma1u14xa:~$ echo -e 'aa\nbb'
aa
bb

> env | grep -i shell
SHELL=/bin/csh
> echo -e 'aa\nbb'
-e aa\nbb
```
It is seems in mac env:
```
% env |grep -i shell
SHELL=/bin/zsh
llhu@huleis-mbp rule-engine % bash
bash-5.0$ echo -e "a\nb"
a
b
```

Reply:
Ah, that makes sense. So i think this PR should be closed. Not that many people use csh anymore. Even if they do, the instructions in the README.md still work, you can still upload `mms-file` to CSS and download it. It's just that the file content looks a little odd.

Reply:
@bmpotter , yea, I am fine to close this, Thanks for your active response.